Source/Particles/MultiParticleContainer.cpp

```diff
@@ -539,11 +539,11 @@ MultiParticleContainer::DepositCurrent (
     const amrex::Real dt, const amrex::Real relative_time)
 {
     // Reset the J arrays
-    for (int lev = 0; lev < J.size(); ++lev)
+    for (auto& J_lv : J)
     {
-        J[lev][0]->setVal(0.0_rt);
-        J[lev][1]->setVal(0.0_rt);
-        J[lev][2]->setVal(0.0_rt);
+        J_lv[0]->setVal(0.0_rt);
+        J_lv[1]->setVal(0.0_rt);
+        J_lv[2]->setVal(0.0_rt);
```

Comment:
Could we call this `J_lev` instead of `J_lv`, so that it is as self-explanatory as possible?